examples/mobile/ScheduleApp/css/style.css

```diff
@@ -81,4 +81,9 @@
 	position: relative;
 	top: 1px;
 	margin: auto;
+}
+
+.content-subheader {
+	height: 52px;
```

Comment:
this seems to be a too big gap according to the GUI (2.1)
![image](https://user-images.githubusercontent.com/2246331/82538604-4a9f6000-9b4c-11ea-9511-a031634c6832.png)
and 
Reference app 
f/[WebApp]Basic_Reference_GUI_V1.0_200311/index.html#artboard9
![image](https://user-images.githubusercontent.com/2246331/82538864-ac5fca00-9b4c-11ea-915e-c8e7f84161be.png)

I wander if we should do the styling here - shouldn't it be a TAU library part?

Reply:
Done!